modules/helm-understanding-helm.adoc

```diff
@@ -6,8 +6,15 @@
 [id="helm-understanding-helm_{context}"]
 = Understanding Helm
 
-Helm is a command-line interface (CLI) tool that simplifies deployment of applications and services to {product-title} clusters. Helm uses a packaging format called _charts_.
-A Helm chart is a collection of files that describes {product-title} resources. A running instance of the chart in a cluster is called a _release_. A new release is created every time a chart is installed on the cluster.
+Helm is a software package manager that simplifies deployment of applications and services to {product-title} clusters.
```

Comment:
s/of applications/of the applications

Reply:
These are generic applications and services and not specific ones so I have skipped the definite article here.